docs/cd_with_aws.md

```diff
@@ -34,7 +37,28 @@ Before you start, make sure you have the following:
      - `ECS_SERVICE`: The name of your ECS service.
      - `ECS_CLUSTER`: The name of your ECS cluster.
 
-6. **GitHub Actions Workflow**:
+6. **Brief Guide to Configure AWS Systems Manager Parameter Store for GitHub Actions Workflow**:
+
+   - **Access AWS Systems Manager**:
+
+     - Log in to your AWS console.
+     - Navigate to **Systems Manager** and select **Parameter Store**.
+
+   - **Create Parameters**:
+
+     - Click **Create parameter**.
+     - Fill out the details:
+       - **Name**: Provide a unique and valid name (e.g., `/rails_api_base/backend/service_api_key`).
```

Comment:
I think we should also clarify what's the expected naming convention here

Reply:
Sure